docs/preview/03-Features/sinks/azure-application-insights.md

```diff
@@ -38,6 +38,18 @@ ILogger logger = new LoggerConfiguration()
     .CreateLogger();
 ```
 
+> ⚡ When you want to initialize Microsoft's `TelemetryClient` directly from the sink, you can pass in the `IServiceProvider` so that the sink registration re-uses your registered `TelemetryClient`.
```

Comment:
Because there are different scenarios that we support where this sink can be used, and not every one need a `TelemetryClient` directly.
You could also opt to, for example, set the connection string via environment variables, so that the `TelemetryClient` is set automatically, and if the Arcus secret store includes environment variables, you can also use that to configure the Serilog sink.
So, there are multiple options and scenarios where you would want or don't want to set it via the Serilog sink, and so it felt a bit restricted and wrong to mark the other ones as deprecated.

Reply:
Ok.  We should indeed not enforce one specific scenario.

Reply:
Maybe simplify that sentence a bit then ?
```suggestion
> ⚡ When you want to re-use an already registered `TelemetryClient`, pass in the `IServiceProvider` to the method that registers the Application Insights sink.
```

Additional question(s):
- In what scenario's would you already have registered a TelemetryClient ?
- What happens if you call the overload that takes the `serviceProvider` parameter, but no `TelemetryClient` is registered ?  Do we throw ?  Or do we instantiate a new instance ?

Reply:
If we recommended using Arcus, you will not see the client registered as it will be done implicitly. Therefore, changing this sentence to your proposal would make it harder as people using all of Arcus would not even be aware that the client is registered.

> In what scenario's would you already have registered a TelemetryClient ?

If you use W3C in HTTP or Messaging, or you use the Serilog sink yourself without any additional Arcus components and register the client yourself.

> What happens if you call the overload that takes the serviceProvider parameter, but no TelemetryClient is registered ? Do we throw ? Or do we instantiate a new instance ?

Answered a bit below.

Reply:
Actually, I don't see why the proposal would confuse people.  I have difficulties getting my head around 'initialize directly from the sink', as I don't know what is really meant with that ?

Reply:
> Actually, I don't see why the proposal would confuse people

Because the `TelemetryClient` is registered implicitly. But we can use your proposal, but we should mention that some Arcus extensions registers this client by default, if using the W3C correlation system.

Reply:
Indeed.  I would then say explicitly that if you use Arcus middleware, you should pass in the serviceprovider, because the Arcus middleware registers the TelemetryClient behind the scenes.

Reply:
Changed to your suggestion and added an additional sentence to explain how the `TelemetryClient` is implicitly registered in Arcus ocmponents.